x-pack/plugins/security_solution/server/lib/detection_engine/rule_response_actions/schedule_notification_response_actions.test.ts

```diff
@@ -14,7 +14,7 @@ import { responseActionsClientMock } from '../../../endpoint/services/actions/cl
 
 describe('ScheduleNotificationResponseActions', () => {
   const signalOne = {
-    agent: { id: 'agent-id-1' },
+    agent: { id: 'agent-id-1', type: 'endpoint' },
```

Comment:
I'm assuming we no longer needs these changes, correct? I don't see a problem keeping them, but just wanted to confirm they are not necessary given the fix that was done

Reply:
You're right, not needed, and probably eslint will throw errors. I wanted to keep them so we know that it accepts different types, but it doesn't makes sense on second thought.